pr-data.csv

```diff
@@ -2990,7 +2990,7 @@ https://github.com/espertechinc/esper,590fa9c9eb854f1420b9d337b802aca19f963cc0,r
 https://github.com/fabiomaffioletti/jsondoc,de5bd0024b4e5444a541fa4fa4ec518176b646f3,jsondoc-core,org.jsondoc.core.scanner.builder.JSONDocApiAuthBuilderTest.testApiAuthToken,ID,Accepted,https://github.com/fabiomaffioletti/jsondoc/pull/261,
 https://github.com/facebook/facebook-java-business-sdk,492cda9aa1bac6e04e1b80de1794e74f9cea1cb6,facebook-java-business-sdk,com.facebook.ads.sdk.serverside.UserDataTest.BuilderAndGettersWithMultipleValuesTest,ID,Opened,https://github.com/facebook/facebook-java-business-sdk/pull/359,
 https://github.com/facebook/facebook-java-business-sdk,492cda9aa1bac6e04e1b80de1794e74f9cea1cb6,facebook-java-business-sdk,com.facebook.ads.ServerSideTest.CustomDataParametersTest,ID,Opened,https://github.com/facebook/facebook-java-business-sdk/pull/360,
-https://github.com/facebook/facebook-java-business-sdk,4d9fd05f8d3909d23093fbd65c6b2b84c32f1ba9,.,com.facebook.ads.ServerSideTest.CustomDataParametersTest,ID,,,
+https://github.com/facebook/facebook-java-business-sdk,4d9fd05f8d3909d23093fbd65c6b2b84c32f1ba9,.,com.facebook.ads.ServerSideTest.CustomDataParametersTest,ID,failWithMvnCompile,,
```

Comment:
There's no status `failWithMvnCompile`, but you can add to notes which Java version you tried and which project version you tried.